server.js

```diff
@@ -10,6 +10,15 @@ const app = express();
 // the __dirname is the current directory from where the script is running
 app.use(express.static(distPath));
 
+app.use(function(req, res, next) {
```

Comment:
is this really needed? I think it can be removed.

Reply:
yeah go ahead